.vscode/settings.json

```diff
@@ -4,5 +4,8 @@
     },
     "rust-analyzer.runnableEnv": {
         "RUST_LOG": "trace"
-    }
+    },
+    "rust-analyzer.linkedProjects": [
+        ".\\examples\\rp2040\\Cargo.toml"
```

Comment:
weird backslash paths? :D 

also this shoudn't be needed if the user opens `workspace.code-workspace`.

Reply:
Oups this was auto generated by vscode